File: Service_eHospital_DongNai_A/Implement/KhamBenh_SanPhuKhoaThaiService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class KhamBenh_SanPhuKhoaThaiService : BaseService<KhamBenh_SanPhuKhoaThai, IKhamBenh_SanPhuKhoaThaiRepository>
    , IKhamBenh_SanPhuKhoaThaiService
    {
    private readonly IKhamBenh_SanPhuKhoaThaiRepository _KhamBenh_SanPhuKhoaThaiRepository;
    public KhamBenh_SanPhuKhoaThaiService(IKhamBenh_SanPhuKhoaThaiRepository KhamBenh_SanPhuKhoaThaiRepository) : base(KhamBenh_SanPhuKhoaThaiRepository)
    {
    _KhamBenh_SanPhuKhoaThaiRepository = KhamBenh_SanPhuKhoaThaiRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/KhamBenh_SanPhuKhoaThai.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { KhamBenh_SanPhuKhoaThai } from './KhamBenh_SanPhuKhoaThai.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class KhamBenh_SanPhuKhoaThaiService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'KhamBenh_SanPhuKhoaThai_Id', 'BenhNhan_Id', 'KhamBenh_Id', 'NgayKinhCuoiDS', 'TienCanBenhLy', 'ChieuCao', 'CanNang', 'NgayTuoiThai', 'HA', 'Phu', 'BCTC', 'ThaiMay', 'TimThay', 'NgoiThai', 'CTC', 'KetLuan', 'NguoiKham_Id', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: KhamBenh_SanPhuKhoaThai[] | undefined;
    ListFilter: KhamBenh_SanPhuKhoaThai[] | undefined;
    FormData!: KhamBenh_SanPhuKhoaThai;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "KhamBenh_SanPhuKhoaThai";
    }
}

